src/routes/files/handlers.js

```diff
@@ -0,0 +1,51 @@
+import { v4 as uuidv4 } from 'uuid';
+import handleErrors from '../../lib/apiErrorHandler';
+import { sequelize, File } from '../../models';
+import s3Uploader from '../../lib/s3Uploader';
+
+const fs = require('fs');
+const fileType = require('file-type');
+const multiparty = require('multiparty');
+
+const namespace = 'SERVICE:FILES';
+
+const logContext = {
+  namespace,
+};
+
+export async function createFileMetaData(originalFileName, s3FileName) {
```

Comment:
I'd probably hold off exporting this function until it is used somewhere other then this file